suggestedimages/static/index.js

```diff
@@ -1,8 +1,25 @@
+function updateLanguage(langElem) {
+    const code = langElem.value;
```

Comment:
I'm starting to annoy myself buuut could the naming be a bit more verbal here?

Reply:
True, it was a bad name. Changed it to languageInput.